src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/files/main.js

```diff
@@ -94,6 +106,9 @@ define([
             // add uploaded medias to datagrid
             this.sandbox.on('husky.dropzone.' + this.options.instanceName + '.success', function(file, mediaResponse) {
                 this.sandbox.emit('sulu.labels.success.show', 'labels.success.media-upload-desc', 'labels.success');
+                mediaResponse = this.sandbox.util.extend(true, mediaResponse, {
```

Comment:
Why has it to be cloned? And I would prefer to use the `deepCopy` method for that.